bedrock/infobar/templates/infobar/infobar.jsonp

```diff
@@ -0,0 +1,14 @@
+{% set_lang_files "tabzilla/tabzilla" %}
+_({
+    {# L10n: You can replace 'in your language' in this string by the name of your language if you want. -#}
+    "message": "{{ _('Would you like to see this page in your language?')|js_escape }}",
+    {# L10n: Your translation can just say Yes -#}
+    "accept": "{{ _('Yes, please!')|js_escape }}",
+    {# L10n: Your translation can just say No -#}
+    "cancel": "{{ _('No, thanks.')|js_escape }}",
+    {# L10n: Update bar messaging -#}
+    "update-message": "{{ _('Looks like you’re using an older version of Firefox.')|js_escape }}",
+    "update-accept": "{{ _('Update to stay fast and safe.')|js_escape }}",
+    "update-cancel": "{{ _('Later')|js_escape }}",
+    "latestfx": "{{ latest_firefox_version }}",
```

Comment:
May be an old rule, but I think to be as broadly supportive of browsers as possible a trailing comma is still bad. I could be wrong though.